### PR TITLE
A few media type fixes (stolen from RFC 8710)

### PR DESCRIPTION
**Pull request by cabo**
The encoding column in RFC 7252 really is a "content-coding" column, which you don't have, hence "-".

Used RFC 8710 to copy some common media-type concerns.
You don't want LIMITED USE, this means you don't want to use this outside a specific experiment.

### CODE REVIEW

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -646,25 +646,37 @@ signed with a COSE_Sign1 structure {{I-D.ietf-cose-rfc8152bis-struct}}.
...
+      ANIMA WG [^ouch]
+    Intended usage:  LIMITED [^ouch2]
     Restrictions on usage:  NONE
     Author:  ANIMA WG
     Change controller:  IETF
     Provisional registration? (standards tree only):  NO
 
+[^ouch]: That will need to be fixed so there is a contact point beyond
+    the time the ANIMA WG exists.  Please see RFC 8710 for an example.
+
```
Comment (mcr) [inline constrained-voucher.mkd:675]: so, please either tell me what to put, or raise an issue.
A pull request polluting the text with a pointer to something to do is rather non-actionable.

Comment (cabo) [reply constrained-voucher.mkd:675]: ART has defined rules for that; I could try to recollect those.  But this is a different area, so I'd ask the AD.

Comment (cabo) [reply constrained-voucher.mkd:675]: See also I-D-to-be https://cabo.github.io/cbor-media-types/draft-bormann-cbor-defining-media-types.html

**Review by mcr — APPROVED** [review]

File: constrained-voucher.mkd [hunk 2]
```diff
@@ -646,25 +646,37 @@ signed with a COSE_Sign1 structure {{I-D.ietf-cose-rfc8152bis-struct}}.
...
+      document, there is no fragment identification syntax defined for
+      application/cbor.)
     Additional information:
       Magic number(s):  None
       File extension(s):  .vch
       Macintosh file type code(s):  none
     Person & email address to contact for further information:  IETF
-      ANIMA WG
-    Intended usage:  LIMITED
+      ANIMA WG [^ouch]
```
Comment (mcr) [inline constrained-voucher.mkd:666]:
```suggestion
      ANIMA Working Group (anima@ietf.org) or IETF Operations and
         Management Area Working Group (opsawg@ietf.org)
```

File: constrained-voucher.mkd (hunk 1 above)
Comment (mcr) [inline constrained-voucher.mkd:676]:
```suggestion

```

**Pull request merged**